mystyle.css

```diff
@@ -32,26 +32,24 @@ nav {
 }
 
 /*List Members Bar style*/
-#listMembersBar {
+.list-members-bar,
```

Comment:
Since I removed that repeated ID from those elements in the HTML, I had to update this CSS rule